### DISCUSSION TITLE
一个新 C++ language server 的设计与实现 · ykiko's blog

### DISCUSSION THREAD

**Discussion by giscus** [16bit-ykiko/about-me#4] [Announcements]
# 一个新 C++ language server 的设计与实现 · ykiko's blog

mainly about computer science and mathematics

https://ykiko.me/zh-cn/articles/13394352064/

**Comment by tsurumi-yizhou** [comment]
是否可以加入一个帮助将基于头文件的代码组织方式迁移到 modules 的功能，虽然可能并不属于 lsp 该干的工作，然而这个功能和业务范畴确实很接近

**Reply by 16bit-ykiko** [reply]
其实已经有这样的工具了: [clang-modules-converter](https://github.com/ChuanqiXu9/clang-modules-converter)
